project-icp/bee-pollinator-app
# Crop Cover Patterns

## Overview

To differentiate cover crops from regular crops, we add an overlay pattern to them. In addition, we add a new script to automate and maintain consistency of colors across the app. This script also has variables to easily change pattern parameters.

Connects #154 

### Demo

![image](https://cloud.githubusercontent.com/assets/1430060/22946338/1fdc29f8-f2c5-11e6-90db-cead65e33aa2.png)

## Testing Instructions

 * Check out the branch, ensure the colors match up in general (across charts, thumbnails, shapes on map)
 * Ensure cover crops have a cross-hatch pattern on them
 * Run `./scripts/colors/update.py` and ensure that no files are changed in git (new files should be identical to old ones)
 * Try changing the stroke width, or line color, or a color in `cdl_colormap`, and run the script again. Ensure the changes carry through. Run bundle, and ensure new colors / patterns are visible in the app.
